### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by arijit1111**
Bazar Admin

### CODE REVIEW

File: Modules/Backend/Http/Controllers/BackendController.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+use Illuminate\Routing\Controller;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Validator;
+use Modules\User\Entities\User;
+use Session;
+
+class BackendController extends Controller
+{
+
+    public function index()
```
Comment (shoaibrehman) [inline Modules/Backend/Http/Controllers/BackendController.php]: phpdocs missing

File: Modules/Backend/Http/Controllers/BackendController.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    {
+        return view('backend::index');
+    }
+
+    public function checkLogin(Request $request)
```
Comment (shoaibrehman) [inline Modules/Backend/Http/Controllers/BackendController.php]: phpdocs missing

File: Modules/Backend/Http/Controllers/BackendController.php [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                    return back()->with('errors', 'Your password is wrong');
+                }
+            } else {
+                return back()->with('errors', 'Email address not found');
+            }
+        }
+
+    }
+
+    public function logOut()
```
Comment (shoaibrehman) [inline Modules/Backend/Http/Controllers/BackendController.php]: phpdocs missing, and remove blank lines

File: Modules/Backend/Http/Controllers/BackendController.php [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    {
+
+        Session::forget('AdminId');
+        return redirect('backend');
+
+    }
+
+    public function changePassword()
```
Comment (shoaibrehman) [inline Modules/Backend/Http/Controllers/BackendController.php]: phpdocs missing

File: Modules/Backend/Http/Controllers/BackendController.php [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    {
+        return view('backend::ChangePassword');
+    }
+
+    public function updatePassword(Request $request)
```
Comment (shoaibrehman) [inline Modules/Backend/Http/Controllers/BackendController.php]: phpdocs missing

File: Modules/Backend/Http/Controllers/DashboardController.php [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+use Illuminate\Routing\Controller;
+use Modules\User\Entities\User;
+use Modules\Product\Entities\Products;
+use Modules\Order\Entities\Order;
+use Session;
+
+class DashboardController extends Controller
+{
+
+    public function index()
```
Comment (shoaibrehman) [inline Modules/Backend/Http/Controllers/DashboardController.php]: phpdocs missing

File: Modules/Backend/Http/Controllers/ProductController.php [hunk 7]
```diff
@@ -0,0 +1,352 @@
...
+use Modules\Product\Entities\ProductVariation;
+use Modules\Product\Entities\Video;
+use Modules\Country\Entities\Country;
+use Session;
+use DB;
+
+class ProductController extends Controller
+{
+
+    public function index(Request $request)
```
Comment (shoaibrehman) [inline Modules/Backend/Http/Controllers/ProductController.php]: phpdocs missing

File: Modules/Backend/Http/Controllers/ProductController.php [hunk 8]
```diff
@@ -0,0 +1,352 @@
...
+    {
+        if (!Session::has('AdminId')) {
+            return redirect('/backend');
+        }
+
+        return view('backend::ProductList', ['brand_id' => $request->brand_id]);
+
+    }
+    public function vendorProduct(Request $request)
```
Comment (shoaibrehman) [inline Modules/Backend/Http/Controllers/ProductController.php:42]: phpdocs missing

File: Modules/Backend/Http/Controllers/ProductController.php [hunk 9]
```diff
@@ -0,0 +1,352 @@
...
+                'website' => $v->website,
+
+            );
+            $slno++;
+        }
+        echo $json_data = json_encode(array("data" => $data));
+        exit;
+    }
+
+    public function productDetail(Request $request)
```
Comment (shoaibrehman) [inline Modules/Backend/Http/Controllers/ProductController.php]: phpdocs missing

File: Modules/Backend/Http/Controllers/VendorController.php [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+use Illuminate\Http\Request;
+use Illuminate\Routing\Controller;
+use DB;
+use Session;
+use Modules\User\Entities\User;
+
+class VendorController extends Controller
+{
+
+    public function index()
```
Comment (shoaibrehman) [inline Modules/Backend/Http/Controllers/VendorController.php]: phpdocs missing

File: Modules/Backend/Http/Controllers/VendorController.php [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+                'logo_image' => $items->logo_image
+
+            );
+            $count++;
+        }
+        echo $json_data = json_encode(array("data" => $data));
+        exit;
+    }
+
+    public function vendorDetails(Request $request)
```
Comment (shoaibrehman) [inline Modules/Backend/Http/Controllers/VendorController.php]: phpdocs missing

File: Modules/Backend/Resources/views/ChangePassword.blade.php [hunk 12]
```diff
@@ -0,0 +1,49 @@
+@include('backend::header')
```
Comment (shoaibrehman) [inline Modules/Backend/Resources/views/ChangePassword.blade.php:1]: Move all inline css into a css file

**Review by shoaibrehman — CHANGES_REQUESTED** [review]

File: Modules/Backend/Http/Controllers/BackendController.php (hunk 1 above)
Comment (arijit1111) [reply Modules/Backend/Http/Controllers/BackendController.php]: Fixred

File: Modules/Backend/Http/Controllers/BackendController.php (hunk 3 above)
Comment (arijit1111) [reply Modules/Backend/Http/Controllers/BackendController.php]: Fixed

File: Modules/Backend/Resources/views/ChangePassword.blade.php (hunk 12 above)
Comment (arijit1111) [reply Modules/Backend/Resources/views/ChangePassword.blade.php:1]: We will include all the inline CSS into main css file after Admin part is done

**Review by shoaibrehman — APPROVED** [review]
PR approved.

**Pull request merged**